docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,469 @@
+
+aws.dynamodb.table_names: ["Users"]
+```
+
+### `aws.sdk.dynamodb.describetable`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+```
+
+### `aws.sdk.dynamodb.getitem`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+aws.dynamodb.projection: "Title, Description, RelatedItems, ProductReviews" // The value of the ProjectionExpression request parameter.
+```
+
+### `aws.sdk.dynamodb.putitem`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+```
+
+### `aws.sdk.dynamodb.query`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+aws.dynamodb.scan_forward: true // The value of the ScanIndexForward request parameter.
+aws.dynamodb.attributes_to_get: ["lives", "id"] // The value of the AttributesToGet request parameter.
+aws.dynamodb.consistent_read: true // The value of the ConsistentRead request parameter.
+aws.dynamodb.index_name: "my_index" // The value of the IndexName request parameter.
+aws.dynamodb.limit: 50 // The value of the Limit request parameter.
+aws.dynamodb.projection: "Title, Description, RelatedItems, ProductReviews" // The value of the ProjectionExpression request parameter.
+aws.dynamodb.select: "ALL_ATTRIBUTES" // The value of the Select request parameter.
+```
+
+
+### `aws.sdk.dynamodb.scan`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+aws.dynamodb.segment: 10 // The value of the Segment request parameter.
+aws.dynamodb.total_segments: 100 // The value of the TotalSegments request parameter.
+aws.dynamodb.count: 5 // The value of the Count request parameter
+aws.dynamodb.scanned_count: 5 // The value of the ScannedCount request parameter.
+aws.dynamodb.attributes_to_get: ["lives", "id"] // The value of the AttributesToGet request parameter.
+aws.dynamodb.consistent_read: true // The value of the ConsistentRead request parameter.
+aws.dynamodb.index_name: "my_index" // The value of the IndexName request parameter.
+aws.dynamodb.limit: 50 // The value of the Limit request parameter.
+aws.dynamodb.projection: "Title, Description, RelatedItems, ProductReviews" // The value of the ProjectionExpression request parameter.
+aws.dynamodb.select: "ALL_ATTRIBUTES" // The value of the Select request parameter.
+```
+
+### `aws.sdk.dynamodb.updateitem`
```

Comment:
@ac360 Why not `updateItem`? We just want `[a-z0-9.]` as the allowed charset for span names?